types/storage_key_test.go

```diff
@@ -60,31 +60,31 @@ func TestCreateStorageKeyPlainV4(t *testing.T) {
 func TestCreateStorageKeyMapV10(t *testing.T) {
```

Comment:
Can revert this file since V10,V9 and V4 contained AccountNonce.

Reply:
reverted